### ISSUE TITLE
Anyone still using this?

### ISSUE THREAD

**Issue by Erudition** [elm-athlete/athlete#30]
I noticed this repo (and github org, with no public members) seems abandoned. The concept seems really cool, would anyone recommend using it in a new project?

I'm specifically concerned about the latest fixes being mostly from 2017, and the website is down, and the project was renamed to athlete with none of the rest of the resources updated to reflect this.

It also seems (from the code) there was a WYSIWYG implementation being built, cool!

**Comment by CharlonTank** [comment]
Hey there! For the time being, it's on hold, but there's a chance it could make a return in the future. I genuinely appreciate the smooth 60fps transitions between routes that `elm-athlete` offers. I actually used it for a while, but then switched to `elm-ui`, even though it's not quite as robust as `elm-athlete`, simply because it receives more frequent updates.

**Comment by CharlonTank** [comment]
Hey there!

At the moment, the project is on hold, but there's still a possibility that it might come back later on. I really appreciate the routing system and smooth page transitions that elm-athlete provides. What's great about using elm-athlete is that it encourages me to dwelve deeper into CSS, unlike elm-ui (obviously haha). And also, unlike elm-css, it's typesafe, so it's easier to understand CSS features.

About the WYSIWYG, I'm not sure.

**Comment by Erudition** [comment]
Yes, even without committing to Athlete's routing and transition system, the type safe css and html api seems really appealing. I just  wish the docs were more complete on how to use it, and for some signs that the project won't stop working soon due to neglect.

**Comment by tibastral** [comment]
Thank you for the kind words Erudition. Yes, it was developed for exactly that purpose : stopping to worry about using css properties that don't apply to your html nodes, and not being able to add html properties that don't apply to the current node too. I'll go back to develop it as soon as possible.
